python/tests/test_drawing.py

```diff
@@ -1435,34 +1441,36 @@ def verify_basic_svg(self, svg, width=200, height=200):
             cls = group.attrib["class"]
             assert re.search(r"\broot\b", cls)
 
-    def test_draw_file(self):
+    def test_draw_to_file(self, tmp_path):
+        # NB: to view output files for testing changes to drawing code, it is possible
+        # to save to a fixed directory using e.g. `pytest --basetemp=/tmp/svgtest ...`
         t = self.get_binary_tree()
-        fd, filename = tempfile.mkstemp(prefix="tskit_viz_")
-        try:
-            os.close(fd)
-            svg = t.draw(path=filename)
-            assert os.path.getsize(filename) > 0
-            with open(filename) as tmp:
-                other_svg = tmp.read()
-            assert svg == other_svg
-            os.unlink(filename)
-
-            svg = t.draw_svg(path=filename)
-            assert os.path.getsize(filename) > 0
-            with open(filename) as tmp:
-                other_svg = tmp.read()
-            self.verify_basic_svg(svg)
-            self.verify_basic_svg(other_svg)
-
-            ts = t.tree_sequence
-            svg = ts.draw_svg(path=filename)
-            assert os.path.getsize(filename) > 0
-            with open(filename) as tmp:
-                other_svg = tmp.read()
-            self.verify_basic_svg(svg)
-            self.verify_basic_svg(other_svg)
-        finally:
-            os.unlink(filename)
+        filename = tmp_path / "tree-draw.svg"
+        # print(f"Saved tree.draw() to {filename}")  # `pytest -rP` shows path to SVG
```

Comment:
I think the comment above makes all these `print`s redundant.

Reply:
Re being used elsewhere, these functions are only used in 2 other files, one of which (test_highlevel.py) is a bit hard-coded so would take some work to convert to fixtures.

Reply:
(print statements removed)